creator-node/src/components/healthCheck/healthCheckController.js

```diff
@@ -111,15 +116,50 @@ const healthCheckController = async (req) => {
     randomBytesToSign
   )
 
-  const { stateMachineQueueLatestJobSuccess } = response
-  if (enforceStateMachineQueueHealth && stateMachineQueueLatestJobSuccess) {
-    response.snapbackMaxLastSuccessfulRunDelayMs =
-      SNAPBACK_MAX_LAST_SUCCESSFUL_RUN_DELAY_MS
-    const delta =
-      Date.now() - new Date(stateMachineQueueLatestJobSuccess).getTime()
-    if (delta > SNAPBACK_MAX_LAST_SUCCESSFUL_RUN_DELAY_MS) {
+  const {
+    latestMonitorStateJobSuccess,
+    latestFindSyncRequestsJobSuccess,
+    latestFindReplicaSetUpdatesJobSuccess
+  } = response
+  if (enforceStateMachineQueueHealth && latestMonitorStateJobSuccess) {
+    // Enforce time since last successful monitor-state job
+    response.monitorStateJobLastSuccessfulRunDelayMs =
```

Comment:
will get hard to find all these fields in health check, thoughts on grouping everything in an object under `response.stateMachineQueues` or smth?

Reply:
changed to group them